clang/lib/Frontend/CompilerInvocation.cpp

```diff
@@ -2597,6 +2597,9 @@ static void ParseLangArgs(LangOptions &Opts, ArgList &Args, InputKind IK,
     }
     Opts.SYCLExplicitSIMD = Args.hasArg(options::OPT_fsycl_esimd);
     Opts.EnableDAEInSpirKernels = Args.hasArg(options::OPT_fenable_sycl_dae);
+    Opts.SYCLValueFitInMaxInt =
```

Comment:
Should the default value be 0 or 1? When the LANGOPT is defined above, the default is set to 1. Over here you return false when neither flag is provided.

Reply:
I intent to set 0.  That change gets lost.  Since we originally want this to be default.  We turn off default just because we see lots of fails.  Now all the tests are passing.  The '1' should be right to do.

Reply:
Right now I think there is inconsistency between default values. Either the default value where the langopt is defined in LangOpts.def needs to be changed to 0 or the third argument in hasFlag should be true on L2602.

Reply:
For now, we want the default to be off (which the flag check will force).  I will make the needed adjustment as requested.